wiki/webserver/ExternalWebserversSQL.md

```diff
@@ -101,3 +101,45 @@ ProxyPassMatch ^/(maps/[^/]*/live/.*) http://127.0.0.1:8100/$1
 > **Important:**<br>
 > The above config is **just an example** and not a complete config you can just copy&paste. You **will** need to adapt it to your setup!
 {: .info .important }
+
+## Caddy with php-fpm
+
+Here is an example for how it could look like on Caddy with php-fpm:
+```caddy
+# Replace this with your own domain.
+# Put http://example.com if you are behind a reverse proxy, that way Caddy will not try to process HTTPS
+example.com {
+    root * /home/container/public
+    # https://caddyserver.com/docs/caddyfile/patterns#php-fpm
+    php_fastcgi unix//run/php/php7.4-fpm.sock # May need to be changed
+    file_server
+    # Rewrite requests to sql.php
+    try_files {path} /sql.php
+}
+```
```

Comment:
I hate to make such an extensive review and rewrite of the code, but some changes needed to be made as well as better documented.

```suggestion
## Caddy with php-fpm

Here is an example for how it could look like on Caddy with php-fpm:
```caddy
# Replace this with your own domain.
# Put http://example.com if you are behind a reverse proxy, that way Caddy will not try to process HTTPS
example.com {
	# This designates the root for the webserver
	root /var/www/map.hasphetica.win
	file_server

	# https://caddyserver.com/docs/caddyfile/patterns#php-fpm
	# You may need to modify this path. Here are some examples;
	# /unix//run/php-fpm/php-fpm.sock
	# /unix//run/php/php8.2-fpm.sock
	php_fastcgi unix//run/php/php7.4-fpm.sock


	# NOT OPTIONAL:
	# This directive tells the webserver to use SQL.PHP, which is the main file that handles PHP requests. You need this for PHP.
	handle {
		try_files {path} /sql.php
	}

	# OPTIONAL:
	# This handles logging. Modify INFO to ERROR if you only want error logs, or remove this if you do not want logging.
	log {
		output file /var/log/caddy/map-access.log
		format console
		level INFO
	}

	# OPTIONAL:
	# Proxy requests for live data to the bluemaps integrated webserver.
	# If you have multiple servers you will need to proxy each map-id to the correct server.
	handle /maps/*/live/* {
		reverse_proxy 127.0.0.1:8100
	}
}
```

Here's what should be noted;
```caddy
	handle {
		try_files {path} /sql.php
	}
```

This by default can be read as handle * {etc}; meaning all requests get handled to try sql.php
```caddy
	handle /maps/*/live/* {
		reverse_proxy 127.0.0.1:8100
	}
```

This specifically wildcards what you see above to try and get live-data from the server.

The documentation was also updated to better match it's nginx and apache counterparts.

It should be suggested that these changes are reflected to frakenphp, though that is not my specialty as I do not use it and thus could not test. However, the code should simply be "copy-paste"